sway/desktop/xdg_shell.c

```diff
@@ -385,6 +401,17 @@ static void handle_map(struct wl_listener *listener, void *data) {
 		view_update_csd_from_client(view, csd);
 	}
 
+	if (config->popup_during_fullscreen == POPUP_LEAVE &&
```

Comment:
Now that this part is generic, is it possible not to duplicate it and move it into `view_map`?